apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Snippets/PublisherClientSnippets.cs

```diff
@@ -43,13 +47,14 @@ public void AddCustomizedPublisherClient()
         string projectId = "projectId";
         string topicId = "topicId";
         var services = new ServiceCollection();
+        var googleCredential = CredentialFactory.FromJson<UserCredential>(UserCredentialFileContents).ToGoogleCredential();
 
         // Sample: AddCustomizedPublisherClient
         TopicName topicName = TopicName.FromProjectTopic(projectId, topicId);
         services.AddPublisherClient(builder =>
         {
             builder.TopicName = topicName;
-            builder.CredentialsPath = "path/to/credentials.json";
+            builder.GoogleCredential = googleCredential;
```

Comment:
Maybe now we should leave a note:
```suggestion
            // An instance of Google.Apis.Auth.OAuth2.GoogleCredential that you can create using Google.Apis.Auth.OAuth2.CredentialFactory and other GoogleCredential factory methods.
            builder.GoogleCredential = googleCredential;
```